scripts/sysman/EXT_EngineActivity.rst

```diff
@@ -0,0 +1,76 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+    s=tags['$s']
+    S=s.upper()
+%>
+:orphan:
+
+.. _ZES_extension_engine_activity:
+
+==========================
+ Engine Activity Extension
+==========================
+
+API
+----
+
+* Functions
+
+    * ${s}EngineGetActivityExt
+
+* Structures
+
+    * ${s}_engine_ext_properties_t
+
+Engine Activity
+~~~~~~~~~~~~~~~~~~
+
+This extension provides capability to user to gather the activity stats for each Virtual Function (VF) associated with engine group.
+Intended use case of the feature is to query utilization stats from Physical Function (PF) interface. However, the extension
+may return error code $X_RESULT_ERROR_UNSUPPORTED_FEATURE if an environment does not support it.
```

Comment:
Just thinking: Should we have ther error as unsupported feature - this may mean that this feature would be supported in the future. 
OR insufficient permissions, which may mean the VF does not have the privileges to retrieve the necessary information.

Reply:
I could also add error return for insufficient permissions here that way implementation can choose to return whichever is more suitable

Reply:
so insufficient permission is actually when "user does not have permissions to perform an operation". in this case it's not about user but VF instance that can't read the value due to environment. so i think might be better to use unsupported feature. adding description to that effect.